pkg/crypto/webkms/wrapkey_util.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webkms
+
+import (
+	"encoding/base64"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto"
+)
+
+// WrapKeyReq serializable WrapKey request.
+type WrapKeyReq struct {
+	CEK       string       `json:"cek,omitempty"`
+	APU       string       `json:"apu,omitempty"`
+	APV       string       `json:"apv,omitempty"`
+	RecPubKey PublicKeyReq `json:"recpubkey,omitempty"`
+	SenderKID string       `json:"senderkid,omitempty"`
+}
+
+// WrapKeyResp serializable WrapKey response.
+type WrapKeyResp struct {
+	WrappedKey RecipientWrappedKeyReq `json:"WrappedKey,omitempty"`
+}
+
+// UnwrapKeyReq serializable UnwrapKey request.
+type UnwrapKeyReq struct {
+	WrappedKey RecipientWrappedKeyReq `json:"WrappedKey,omitempty"`
+	SenderKID  string                 `json:"senderkid,omitempty"`
+}
+
+// UnwrapKeyResp serializable UnwrapKey response.
+type UnwrapKeyResp struct {
+	Key string `json:"key,omitempty"`
+}
+
+// RecipientWrappedKeyReq contains recipient key material required to unwrap CEK for HTTP requests.
+type RecipientWrappedKeyReq struct {
+	KID          string       `json:"kid,omitempty"`
+	EncryptedCEK string       `json:"encryptedcek,omitempty"`
+	EPK          PublicKeyReq `json:"epk,omitempty"`
+	Alg          string       `json:"alg,omitempty"`
+	APU          string       `json:"apu,omitempty"`
+	APV          string       `json:"apv,omitempty"`
+}
+
+// PublicKeyReq mainly to exchange EPK in RecipientWrappedKey for HTTP requests.
+type PublicKeyReq struct {
+	KID   string `json:"kid,omitempty"`
+	X     string `json:"x,omitempty"`
+	Y     string `json:"y,omitempty"`
+	Curve string `json:"curve,omitempty"`
+	Type  string `json:"type,omitempty"`
+}
+
+// PubKeyToSerializableReq converts recPubKey into a serializable PublicKeyReq.
+func PubKeyToSerializableReq(recPubKey *crypto.PublicKey) PublicKeyReq {
```

Comment:
I know that we discussed yesterday about making utility functions public but during updating Key Server I didn't find them useful - that functionality doesn't worth additional dependency. Can you consider making these utility methods unexported?

Reply:
sure will do

Reply:
done